extensions/devservices/keycloak/deployment/src/main/resources/dev-templates/provider.html

```diff
@@ -0,0 +1,53 @@
+{#include main}
+{#style}
+.testServiceIcon:hover {
+    color: #3366ac !important;
+    cursor: pointer;
+}
+{/style}
+
+{#script}
+function testService(keycloakUrl){
+    $.post("testService",
+        {
+          keycloakUrl: keycloakUrl,
+          serviceUrl: "http://localhost:8080/api/users/me",
+          realm: "quarkus",
+          client: "quarkus-app",
+          user: "alice"
+        },
+        function(data, status){
```

Comment:
@phillip-kruger Hi Phillip, how to set correctly the value of the result field with the id `result` ? I've tried `$('#result').prop(data);` and just `$('#result').prop("OK");` without the `if` precondition.

Reply:
the element with id `result` is an input, so maybe try `$('#result').val(data);`

Reply:
PS: I have not tried it, so might not work

Reply:
Thanks Phillip, I'll try it

Reply:
@phillip-kruger Yes it works, thanks